tests/test-admin.php

```diff
@@ -410,19 +410,42 @@ public function test_admin_notices_heartbeat_never() {
 	public function test_admin_notices_heartbeat_last_seen() {
 		$this->expectOutputRegex( '/The Elasticsearch server was last seen/' );
 		wp_set_current_user( $this->factory->user->create( array( 'role' => 'administrator' ) ) );
-		update_option( 'sp_heartbeat', time() - SP_Heartbeat()->thresholds['alert'] );
-		SP_Heartbeat()->get_last_beat( true );
+		SP_Heartbeat()->record_pulse( time() - SP_Heartbeat()->thresholds['alert'] - 10 );
 		SP_Admin()->admin_notices();
 	}
 
 	public function test_admin_notices_heartbeat_shutdown() {
 		$this->expectOutputRegex( '/SearchPress has deactivated itself/' );
 		wp_set_current_user( $this->factory->user->create( array( 'role' => 'administrator' ) ) );
-		update_option( 'sp_heartbeat', time() - SP_Heartbeat()->thresholds['shutdown'] );
-		SP_Heartbeat()->get_last_beat( true );
+		SP_Heartbeat()->record_pulse( time() - SP_Heartbeat()->thresholds['shutdown'] - 10 );
 		SP_Admin()->admin_notices();
 	}
 
+	/**
+	 * @group stale-heartbeat
+	 */
+	public function test_admin_notices_heartbeat_stale() {
+		wp_set_current_user( $this->factory->user->create( array( 'role' => 'administrator' ) ) );
```

Comment:
In other places in this CR, new syntax is being used.
```suggestion
		wp_set_current_user( $this->factory->user->create( [ 'role' => 'administrator' ) ] );
```